components/dropdown/dropdown-opener-mixin.js

```diff
@@ -133,6 +267,68 @@ export const DropdownOpenerMixin = superclass => class extends superclass {
 		}
 		opener.setAttribute('aria-expanded', 'true');
 		opener.setAttribute('active', 'true');
+		this._isFading = false;
+	}
+
+	/* used by open-on-hover option */
+	async __onOpenerMouseEnter() {
+		// do not respond to hover events on mobile screens
+		const dropdownContent = this.__getContentElement();
+		if (dropdownContent._useMobileStyling) return;
+		clearTimeout(this._dismissTimerId);
+		if (!this._isOpen) await this.openDropdown(false);
+		this._closeTimerStop();
+		this._isHovering = true;
+	}
+
+	/* used by open-on-hover option */
+	async __onOpenerMouseLeave() {
+		// do not respond to hover events on mobile screens
+		const dropdownContent = this.__getContentElement();
+		if (dropdownContent._useMobileStyling) return;
+		this._isHovering = false;
+		if (this._isOpenedViaClick) return;
+		//Wait before closing so we don't lose hover when we jump from opener to card
+		clearTimeout(this._dismissTimerId);
+		await this.closeDropdown(true);
+	}
+
+	/* used by open-on-hover option */
+	__onOpenerTouch(e) {
+		//Prevents touch from triggering mouseover/hover behavior
+		e.preventDefault();
+		this._closeTimerStop();
+		if (this._isOpen) this.closeDropdown();
+		this._isOpenedViaClick = true;
+		if (!this._isOpen) this.openDropdown(true);
 	}
 
+	/* used by open-on-hover option */
+	_closeTimerStart() {
+		if (this._isOpen) return;
+		clearTimeout(this._setTimeoutId);
+		this._isFading = true;
+		this._setTimeoutId = setTimeout(() => {
```

Comment:
Is the purpose of this `setTimeout` to match the opacity transition time, so that we don't actually `display: none` on the content until the animation completes?

Reply:
That's correct. If we did have the display: none, it would be impossible to re-open the dropdown via hover, since it would disappear from the screen. This way, users have 400ms to hover to the dropdown before it closes.

Reply:
Normally we would listen for the `transitionend` (and `transitioncancel`/`transitionrun` when necessary) event to be in sync with the duration (and perhaps delay) that is specified in the CSS.  Note that if the user prefers reduced motion and it's closed immediately, the timing specified here would need to be different as well I think - another consideration.

Reply:
A follow-up thought on this - if there is no animation and the content is hidden immediately, does that present an issue when the user moves their mouse from the opener to the content?

Reply:
My idea for prefer-reduced-motion is actually to keep the content open for the 400ms, just without the fade-out. After those 400ms, it will close with no transition. That will stop the issue of hiding the content immediately and gives the user time to hover to the content. What are your thoughts on that approach?

Reply:
I think that's fine - it sounds like we will need a timeout no matter what to prevent it from closing immediately.  If it were not for that, I would recommend leveraging the events to tie into the animation.

I might be inclined to add a comment in the two spots where the 400ms is specified - "matches setTimeout" and "matches CSS" so if anyone goes to change either one, they know that they need to change the other as well.